deck-functional-tests/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Does this need to be so precise and decoupled from deck?
Is there a way to tie deck together?

Reply:
Offline I propose perhaps this should be in deck repo so versions align. Can have a separate feature branch for now if it isnt "stable enough" to be in master itself. I dont know that it's a good idea, but seems like a better long term fit given its coupling and scoping to deck.